docs/source/stubs.rst

```diff
@@ -524,6 +524,17 @@ Type Stub Content
 This section documents best practices on what elements to include or
 leave out of type stubs.
 
+Modules excluded fom stubs
+--------------------------
+
+Not all modules should be included into stubs.
+
+It is recommended to exclude:
+
+1. Implementation details, with `multiprocessing/popen_spawn_win32.py <https://github.com/python/cpython/blob/main/Lib/multiprocessing/popen_spawn_win32.py>`_ as a notable example
+2. Modules that are not supposed to be imported, such as ``__main__.py``
+3. Protected modules that start with a single ``_`` char. However, when needed protected modules can still be added
```

Comment:
What does "when needed" mean? We should probably refer to the "Undocumented Objects" section below.

Reply:
Ping @sobolevn

Reply:
Thanks! Updated!